gridstatus/miso.py

```diff
@@ -126,6 +127,12 @@ def _get_load_and_forecast_data(self, verbose=False):
         r = self._get_json(url, verbose=verbose)
         return r
 
+    @lmp_config(
+        supports={
+            Markets.REAL_TIME_5_MIN: ["latest"],
+            Markets.DAY_AHEAD_HOURLY: ["latest"],
+        },
+    )
```

Comment:
once we have this decorator, should we removed logic such as the following in all these files?
```
        if date != "latest":
            raise NotSupported()
```

Reply:
Yes